frontend/src/features/userProfile/state/userProfileActions.ts

```diff
@@ -0,0 +1,78 @@
+import { ApiAction, ApiActionThunk, CALL_API } from "../../../store/api";
+
+import { NewSearchResult } from '../../../qri/search'
+import { UserProfileParams } from '../../../qri/userProfile'
+
+export const searchPageSizeDefault = 25
+
+export function loadUserProfile(username: string): ApiActionThunk {
+  return async (dispatch, getState) => {
+    return dispatch(fetchUserProfile(username))
+  }
+}
+
+export function loadUserProfileDatasets(username: string, userProfileParams: UserProfileParams): ApiActionThunk {
+  return async (dispatch, getState) => {
+    return dispatch(fetchUserProfileDatasets(username, userProfileParams))
```

Comment:
The name `UserProfileParams` implies that the results of the fetch would be user profiles. Do we have a `DatasetListParams`, or a more general `ListParams` that can be used?

Reply:
`UserProfileParams` is meant to say "these are the valid url query params for the `UserProfile` view/route.  For now both lists on this view allow for the same params, but these will probably diverge, so I think we will need to be more specific on these and have a unique set of params for each list (or perhaps extend a generic `ListParams`)

Reply:
I'll change this to `UserProfileDatasetListParams`